test/test_mps.py

```diff
@@ -10070,7 +10070,7 @@ def test_mps_compat(self):
         # If this test is successful, that means that all operations in the comparison logic are supported natively on
         # the MPS backend. Please remove this test as well as the compatibility logic in
         # torch.testing._comparison.TensorLikePair._equalize_attributes
-        actual = torch.tensor(1.0, device="mps")
+        actual = torch.zeros(2, 3, 4, 5, device="mps")
```

Comment:
@kulinseth I've increased the shape to 4 dimensions here, because otherwise this test would pass although `torch.testing.assert_close` is not ready.

Reply:
Sounds good